task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -111,47 +134,84 @@ spec:
           exit 0
         fi
 
+
         if [ -z "$(ls -A .$conffolder)" ]; then
           echo "$conffolder is missing catalog file."
           TEST_OUTPUT="$(make_result_json -r ERROR)"
           echo "${TEST_OUTPUT}" | tee "$(results.TEST_OUTPUT.path)"
           popd
           exit 0
         fi
-        # copy content of conffolder to confdir - will be used in next task - related image check
-        cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
 
-        # We have totally 4 checks here currently
-        check_num=4
+        # copy content of conffolder to confdir - will be used in next task -
+        related image check
+
+        cp -r .$conffolder/*
+        $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
+
+
+        EXTRACT_DIR="/extracted_base_img"
+
+        mkdir "${EXTRACT_DIR}"
+
+        oc image extract ${BASE_IMAGE} --path /:"${EXTRACT_DIR}"
```

Comment:
I think we should handle possible failures here in much the same way as it's done when extracting the conffolder [here](https://github.com/redhat-appstudio/build-definitions/blob/49ec9460ea5248229d89fad18814a36c4d1be32d/task/fbc-validation/0.1/fbc-validation.yaml#L105-L112).